docs/journalist/ending_session.rst

```diff
@@ -6,6 +6,6 @@ Ending your session
 When you are finished using SecureDrop Workstation, close the SecureDrop Client window and shut the computer down completely. This is to take advantage of the protections of full-disk encryption, and to avoid unauthorized access to the Workstation and the files and materials on it, which include any messages and submissions that you have downloaded.
 
 To shut down the computer, click your username in the top righthand corner of
-your screen, and select **Shut Down** from the context menu. Or, in the Qubes
-Applications menu, scroll to the bottom of the menu and select **Log Out > Shut
-Down**.
+your screen, and select **Shut Down** from the menu.
+
+.. | screenshot_shut_down |
```

Comment:
The associated image is missing but that isn't related to changes in this PR.

Reply:
Ah good flag, let me push a quick fix. Ro did take a screenshot but it was of the lefthand menu we don't generally want to recommend (because it includes the session-saving "Log out" option). I'll remove the reference for now.

Reply:
Done.